pkg/vex/vex.go

```diff
@@ -169,12 +169,39 @@ func (vexDoc *VEX) ToJSON(w io.Writer) error {
 	return nil
 }
 
+// EffectiveStatement returns the latest VEX statement for a given product, that
+// is the statement that contains the latest data about impact to a given product.
+func (vexDoc *VEX) EffectiveStatement(product, vulnID string) (s *Statement) {
```

Comment:
I'm wondering if it'd be simpler to start by calling `SortStatements` to do the time sort.

And then you can go latest-to-oldest, doing the "does this statement refer to the given product?" check, and return the statement the first time the answer is "yes". WDYT?

Reply:
OK, changed :+1: